src/picosvg/svg_pathops.py

```diff
@@ -190,3 +190,11 @@ def stroke(
 
 def bounding_box(svg_cmds: SVGCommandSeq):
     return skia_path(svg_cmds, fill_rule="nonzero").bounds
+
+
+def path_area(svg_cmds: SVGCommandSeq, fill_rule: str, simplified=True) -> float:
```

Comment:
You kind of have to dig to find out what simplify means. Suggest comment or just no option, always simplify before computing area.

Reply:
I removed the `simplified` option